tests/unit/data/mongodb_config_args.txt

```diff
@@ -0,0 +1,73 @@
+net,
```

Comment:
Suggest removing all the commas from end of lines.
See comment on the test for context.